docs/SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,92 @@
+Supported Platforms
+===================
+Telegraf is released on Linux, FreeBSD, Windows, and macOS. This doc helps define which distributions and releases are targeted.
+
+Linux
+-----
+Telegraf intent: *Support latest versions of the most popular distributions*
+
+Telegraf supports RHEL, Fedora, Debian, and Ubuntu. InfluxData provides package repositories for these distributions. Instructions for using the package repositories can be found on [docs.influxdata.com](https://docs.influxdata.com/telegraf/v1.16/introduction/installation/)
+
+https://distrowatch.com/dwres.php?resource=major
```

Comment:
Should we add a paragraph mentioning that telegraf _might_ also work for older versions and other distributions? I mean I almost never use the distribution version of telegraf but download the static binary from InfluxData directly instead.